.gitignore

```diff
@@ -1,5 +1,5 @@
+lib-test
 dist
-lib
```

Comment:
This means that we will always checking compiled lib file to github?  

In Vega-Lite and CompassQL, we ignore the compiled folder (`build`, which is equivalent to `lib` here), and then have a publish script that included the compiled `lib` when we publish to npm and bower (via github release).